ros2_modulo_control/Dockerfile

```
@@ -17,6 +21,34 @@ RUN git clone -b galactic --depth 1 https://github.com/ros2/rcl_interfaces.git
 RUN git clone -b galactic --depth 1 https://github.com/ros2/test_interface_files.git
 RUN git clone -b ros2 --depth 1 https://github.com/ros/angles.git
 
+
+FROM ${BASE_IMAGE}:galactic as ros2-control-version-galactic
+COPY --from=ros2-control-sources-galactic ${ROS2_WORKSPACE}/src/ros2_control/ ${ROS2_WORKSPACE}/src/
+
+
+FROM ${BASE_IMAGE}:galactic-devel as ros2-control-version-galactic-devel
+COPY --from=ros2-control-sources-galactic ${ROS2_WORKSPACE}/src/ros2_control/ ${ROS2_WORKSPACE}/src/
+
+
+FROM ${BASE_IMAGE}:humble as ros2-control-version-humble-devel
```

This should include `-devel` I think
```suggestion
ARG BASE_IMAGE
FROM ${BASE_IMAGE}:humble-devel as ros2-control-version-humble-devel
```